templates/ntp.conf.j2

```diff
@@ -13,6 +13,13 @@ filegen loopstats file loopstats type day enable
 filegen peerstats file peerstats type day enable
 filegen clockstats file clockstats type day enable
 
+{% if ntp_tinker_panic is sameas true %}
+# Always reset the clock, even if the new time is more than 1000s away
+# from the current system time. Usefull for VMs that can be paused
+# and much later resumed.
+tinker panic 0
```

Comment:
Couldn't find much on it, but now that #72 is merged, and chrony is also in here, I was wondering if it was important to set `tinker` on `chrony.conf`, but it looks like [maybe it's not necessary](https://www.reddit.com/r/redhat/comments/bzhxph/does_chrony_need_the_tinker_panic_0_config_on_a_vm/eqt1hgq), so I won't worry about it here.